bodhi/server/config.py

```diff
@@ -26,18 +26,13 @@
 def get_configfile():
     configfile = None
     setupdir = os.path.join(os.path.dirname(os.path.dirname(__file__)), '..')
-    if configfile:
```

Comment:
I didn't notice how hiliarious this code used to be when I first reviewed this PR ☺